test/check-system-info

```diff
@@ -82,7 +82,7 @@ class TestSystemInfo(MachineCase):
         b = self.browser
 
         network_time_prefix = "NTP enabled"
-        if m.image == "fedora-23" or m.image == "fedora-testing":
+        if m.image in {"fedora-atomic", "fedora-23", "fedora-testing"}:
```

Comment:
What about doing it like this: https://github.com/mvollmer/cockpit/commit/ce81ae0a28142422656c4a5dc1bec73b6178695d